gui/storage/views.py

```diff
@@ -963,11 +963,26 @@ def volume_lock(request, object_id):
         _n = notifier()
         if '__confirm' not in request.POST and not _n.is_freenas() and _n.failover_licensed():
             remaining_volumes = [v for v in models.Volume.objects.exclude(pk=object_id) if v.is_decrypted()]
+            with client as c:
+                running_jails = c.call('jail.query', [('state', '=', 'up')])
```

Comment:
Is there a way to check if the jail is in said volume?

Reply:
Yeah, we can do that